inference-engine/thirdparty/movidius/XLink/pc/protocols/usb_boot.c

```diff
@@ -123,9 +125,6 @@ static const char *get_pid_name(int pid)
             return supportedDevices[i].name;
     }
 
-    if(usb_loglevel)
-        fprintf(stderr, "%s(): Error pid:=%i not supported\n", __func__, pid);
-
```

Comment:
You just removed logging here instead of replace

Reply:
Yes, because this may be the reason for the extra logs. And it is handled [here](https://github.com/openvinotoolkit/openvino/pull/552/files/f62ded86b0e4c3c6451a21cce6e754d6769149e9#diff-9cb3a95be441ec664312f5bdfd1f3c82L226)